File: src/Accounting/Vns.Erp.Core.Accounting.VbAccountingGUI/Asset/CT_TSCD.vb
Imports Vns.Erp.Core.Admin.Service.Interface
Imports Vns.Erp.Core.Accounting.Report.Service.Interface

Public Class CT_TSCD
#Region "Property"
    Private _DmTscdService As IDmTscdService
    Public Property DmTscdService As IDmTscdService
        Get
            Return _DmTscdService
        End Get
        Set(ByVal value As IDmTscdService)
            _DmTscdService = value
        End Set
    End Property

    Private _RpTheTscdS23DnService As IRpTheTscdS23DnService
    Public Property RpTheTscdS23DnService As IRpTheTscdS23DnService
        Get
            Return _RpTheTscdS23DnService
        End Get
        Set(ByVal value As IRpTheTscdS23DnService)
            _RpTheTscdS23DnService = value
        End Set
    End Property
#End Region

    Public Function Bindata(ByVal TSCD_ID As Guid)
        Return _RpTheTscdS23DnService.GetData(TSCD_ID)
    End Function
End Class

File: src/Asset/Vns.Erp.Core.Asset.CsAssetGUI/Asset/CT_TSCD.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using Vns.Erp.Core.Asset.Service.Interface;
using Vns.Erp.Core.Accounting.Report.Service.Interface;

namespace Vns.Erp.Core.Asset.CsAssetGUI
{
    public partial class CT_TSCD 
    {
        #region "Property"
        private IDmTscdService _DmTscdService;
        public IDmTscdService DmTscdService
        {
            get { return _DmTscdService; }
            set { _DmTscdService = value; }
        }

        private IRpTheTscdS23DnService _RpTheTscdS23DnService;
        public IRpTheTscdS23DnService RpTheTscdS23DnService
        {
            get { return _RpTheTscdS23DnService; }
            set { _RpTheTscdS23DnService = value; }
        }
        #endregion

        public object Bindata(Guid TSCD_ID)
        {
            return _RpTheTscdS23DnService.GetData(TSCD_ID);
        }
    }
}